pkg/koordlet/metricsadvisor/collectors/performance/performance_collector_linux.go

```diff
@@ -79,8 +84,22 @@ func (p *performanceCollector) Run(stopCh <-chan struct{}) {
 	if features.DefaultKoordletFeatureGate.Enabled(features.PSICollector) {
 		p.collectPSI(stopCh)
 	}
-	if features.DefaultKoordletFeatureGate.Enabled(features.CPICollector) {
-		go wait.Until(p.collectContainerCPI, p.cpiCollectInterval, stopCh)
+	if p.EnabledPerf() {
```

Comment:
Is it possible that we provide two utils: one is using perf-util, the other is using libpfm4.
This could be an option, both for comparing and stability in production env.

Reply:
Are you suggesting adding the `libpfm4` approach while retaining the existing CPI calculation method?

Reply:
yes, it could be an args, perf/libpfm.